react/modules/debouncedPixelHook.ts

```diff
@@ -1,10 +1,10 @@
 import { useRef } from 'react'
-import { usePixel } from 'vtex.pixel-manager/PixelContext'
+import { usePixel } from 'vtex.pixel-manager'
```

Comment:
Is there any risk here related with the add-to-cart-button problem?

Reply:
No! Not anymore :)

The root of that problem was solved over at pixel-manager.